cfgov/scripts/http_smoke_test.py

```diff
@@ -88,6 +97,9 @@
     '/policy-compliance/enforcement/',
     '/policy-compliance/notice-opportunities-comment/',
     '/policy-compliance/amicus/',
+    '/policy-compliance/guidance/implementation-guidance/hmda-implementation/',
+    '/policy-compliance/guidance/implementation-guidance/mortserv/',
+    '/policy-compliance/guidance/implementation-guidance/tila-respa-disclosure-rule/',
```

Comment:
`flake8` will ping you with some of these long lines. You might need `# noqa: E501` at the end to ignore that.